merger_io.go

```diff
@@ -276,13 +276,13 @@ func (s *ForkAwareDStoreIO) DeleteForkedBlocksAsync(inclusiveLowBoundary, inclus
 		}
 		obf := bstream.MustNewOneBlockFile(filename)
 		if obf.Num > inclusiveHighBoundary {
-			return io.EOF
+			return dstore.StopIteration
 		}
 		forkedBlockFiles = append(forkedBlockFiles, obf)
 		return nil
 	})
 
-	if err != nil && err != io.EOF {
+	if err != nil && err != dstore.StopIteration {
```

Comment:
```suggestion
	if err != nil {
```

Using `dstore.StopIteration` removes the need to check the error manually, the implementation is going to return `nil` (unless you tell me it didn't work as expected?)